cdap-ui/app/features/admin/controllers/namespace/create-ctrl.js

```diff
@@ -33,6 +32,11 @@ angular.module(PKG.name + '.feature.admin')
             myNamespace.getList(true).then(function() {
               EventPipe.emit('namespace.update');
             });
+
+            $timeout(function () {
+              $modalInstance.close();
```

Comment:
Is ```$modalInstance.dismisAll()``` not working? is ```$timeout``` the only solution? Seems like a hack. Does the docs suggest anything?